web/src/pm_list_data.js

```diff
@@ -81,9 +83,17 @@ export function get_conversations() {
 
 // Designed to closely match topic_list_data.get_list_info().
 export function get_list_info(zoomed) {
-    const conversations = get_conversations();
+    let conversations = get_conversations();
 
     if (zoomed || conversations.length <= max_conversations_to_show) {
+        if (zoomed) {
+            const search_term = pm_list.get_pm_search_term();
```

Comment:
The caller should pass the `search_term` in to avoid an import cycle; we just fixed this in the stream list version of this code.